test/e2e/conftest.py

```diff
@@ -145,7 +146,12 @@ def worker_config(
         ), f"Expected exactly one Worker agent whl path, but got {resolved_whl_paths} (from pattern {worker_agent_whl_path})"
         resolved_whl_path = resolved_whl_paths[0]
 
-        dest_path = posixpath.join("/tmp", os.path.basename(resolved_whl_path))
+        if operating_system.name == "AL2023":
+            dest_path = posixpath.join("/tmp", os.path.basename(resolved_whl_path))
+        else:
+            dest_path = posixpath.join(
+                "%USERPROFILE%\\AppData\\Local\\Temp", os.path.basename(resolved_whl_path)
```

Comment:
Does this work for you? I found I had to use `$env:USERPROFILE` see my changes here https://github.com/aws-deadline/deadline-cloud-test-fixtures/pull/125/files. Looks like our only test here is on a Linux worker.

Reply:
Yes, I confirmed this worked with my local whl file to test the `SvcShutdown` change.

Reply:
ok, interesting.